test/index.ts

```diff
@@ -1,38 +1,38 @@
-// @flow
-
 import { describe, it } from 'mocha'
 import assert from 'power-assert'
-import powerCrypt from '../src/index.js'
-
+import powerCrypt from '../src'
 describe('powerCrypt', () => {
   it('crypts string, default encoding is base64', () => {
     const crypted = powerCrypt('shinout')
     assert(typeof crypted === 'string')
   })
-
   it('crypts string with salt', () => {
-    const crypted = powerCrypt('shinout', { salt: 'xx' })
+    const crypted = powerCrypt('shinout', {
+      salt: 'xx',
+    })
     const cryptedWithoutSalt = powerCrypt('shinout')
     assert(crypted !== cryptedWithoutSalt)
   })
-
   it('crypts string with streching', () => {
-    const crypted = powerCrypt('shinout', { nStretch: 400 })
+    const crypted = powerCrypt('shinout', {
+      nStretch: 400,
+    })
     const cryptedWith1000Streaches = powerCrypt('shinout')
     assert(crypted !== cryptedWith1000Streaches)
```

Comment:
variable name is inappropriate and it has a typo
```suggestion
    assert(crypted !== cryptedWith400Stretches)
```